mkt/abuse/serializers.py

```diff
@@ -52,3 +54,13 @@ class WebsiteAbuseSerializer(BaseAbuseSerializer):
 
     class Meta(BaseAbuseSerializer.Meta):
         fields = BaseAbuseSerializer.Meta.fields + ('website',)
+
+
+class ExtensionAbuseSerializer(BaseAbuseSerializer):
+    extension = SplitField(
+        SlugOrPrimaryKeyRelatedField(source='extension', slug_field='slug',
+                                     queryset=Extension.objects.all()),
```

Comment:
Can you log an abuse report for a deleted <thing> ?  Also, no slugs for deleted, is that a problem?  I had serializers.PrimaryKeyRelatedField at first btw - I think Fireplace just uses the pk.